### ISSUE TITLE
Can't access anime details after making a change

### ISSUE THREAD

**Issue by astrotee** [Drutol/MALClient#184]
I can't open the anime details after changing something about it like updating the watched episodes or it's status 
I've to reload the page before trying to open it
This happens on Android idk about the windows version

**Comment by Drutol** [comment]
Hmm I can't really reproduce this. It may be related to MAL IP blocking. Is it persistent?

Sorry for late response I was/am on hiatus.

**Comment by astrotee** [comment]
> Sorry for late response I was/am on hiatus.

no problem ^^

> Is it persistent?

yes it has been since I started using it on Android(about a year ago)
maybe I didn't explain it good enough i'll try to update the issue and tell me if u want more details

**Comment by Drutol** [comment]
So basically clicking on item after incrementing episodes for example does nothing?

Does it behave in the same way on different view modes (detailed/compact list)?

**Comment by astrotee** [comment]
So basically clicking on item after incrementing episodes for example does nothing?


Yes

Does it behave in the same way on different view modes (detailed/compact list)?


I tried it and it only happens with grid

**Comment by Drutol** [comment]
On Thursday I should be finished with exams for this semester I'll investigate it then.

**Comment by Drutol** [comment]
Okay, does disabling swipe to increment or decrement change anything?

**Comment by astrotee** [comment]
Yeah It works when changing from the small menu the status and watched episodes
Guess the problem is just with the swipe

**Comment by Drutol** [comment]
Then for some reason swiping library thinks that you are still swiping, strange strange.
